Core/MIPS/ARM64/Arm64IRCompALU.cpp

```diff
@@ -50,8 +50,13 @@ void Arm64JitBackend::CompIR_Arith(IRInst inst) {
 
 	switch (inst.op) {
 	case IROp::Add:
+		regs_.Map(inst);
+		ADD(regs_.R(inst.dest), regs_.R(inst.src1), regs_.R(inst.src2));
+		break;
```

Comment:
It occurs to me that some of these ops are so simple now that they could be replaced by a lookup table just to an emit-instruction function... Though probably not worth bothering with.

Reply:
Well, they take different args so each would have to be a lambda or something.  Gets messy, not sure it'd be cleaner than a switch in the end.

-[Unknown]